### ISSUE TITLE
force_unicode - py3 compatibility

### ISSUE THREAD

**Issue by nordbit** [geex-arts/jet#5]
```
'jet_tags' is not a valid tag library: ImportError raised loading jet.templatetags.jet_tags: cannot import name 'force_unicode'
```

http://django.readthedocs.org/en/latest/topics/python3.html

**Comment by mightygraf** [comment]
py3 compatibility is top priority now, will be fixed soon
thanks for your interest

**Comment by mightygraf** [comment]
Py3 issues should be fixed in 0.0.4 release
